multiqc/modules/interop/interop.py

```diff
@@ -195,137 +206,198 @@ def run_metrics_summary_table(self, data):
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd',
+            'hidden': True
         }
         headers['Error (100)'] = {
             'title': 'Error Rate 100 Cycles (%)',
-            'description': 'The calculated error rate for cycles 1-100.'
+            'description': 'The calculated error rate for cycles 1-100.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd',
+            'hidden': True
         }
         headers['Intensity C1'] = {
             'title': 'Intensity Cycle 1',
             'description': 'The corresponding intensity statistic at cycle 20 as a percentage of that value at the first cycle.',
             'help': 'equation: 100%x(Intensity at cycle 20)/(Intensity at cycle 1).'
         }
+        headers['%>=Q30'] = {
+            'title': '%>=Q30',
+            'description': 'The percentage of bases with a quality score of 30 or higher, respectively.',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'RdYlGn'
+        }
         table_config = {
             'namespace': 'interop',
             'id': 'interop-runmetrics-detail-table',
             'table_title': 'Sequencing Lane Statistics',
-            'col1_header': '',
-            'no_beeswarm': True
+            'col1_header': 'Run - Lane - Read',
         }
-        return table.plot(data, headers, table_config)
+
+        tdata = {}
+        for s_name in data:
+            for key in data[s_name]['details']:
+                tdata["{} - {}".format(s_name,key)]=data[s_name]['details'][key]
+
+        return table.plot(tdata, headers, table_config)
 
     def index_metrics_summary_table(self,data):
         headers = OrderedDict()
         headers['Total Reads'] = {
             'title': 'Total Reads',
-            'description': 'The total number of reads for this lane.'
+            'description': 'The total number of reads for this lane (in millions).',
+            'modify': lambda x: float(x) / 1000000,
+            'suffix': 'M',
         }
         headers['PF Reads'] = {
             'title': 'PF Reads',
-            'description': 'The total number of passing filter reads for this lane.'
+            'description': 'The total number of passing filter reads for this lane (in millions).',
+            'modify': lambda x: float(x) / 1000000,
+            'suffix': 'M',
         }
         headers['% Read Identified (PF)'] = {
             'title': '% Reads Identified (PF)',
-            'description': 'The total fraction of passing filter reads assigned to an index.'
+            'description': 'The total fraction of passing filter reads assigned to an index.',
+            'suffix': '%',
         }
         headers['CV'] = {
             'title': 'CV',
             'description': 'The coefficient of variation for the number of counts across all indexes.'
```

Comment:
You're missing a comma on these lines, which is triggering the syntax error failures on Travis. Just add a comma at the end of the `'description'` and remove it after `'format'`. eg:

```python
headers['CV'] = {
    'title': 'CV',
    'description': 'The coefficient of variation for the number of counts across all indexes.',
    'format': '{:.,2f}'
}
```